relayer/ops/localenv/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+)
+
+var (
+	namespace string
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		panic("missing required command")
+	}
+
+	switch strings.ToLower(os.Args[1]) {
+	// create k8s cluster + resources
+	case "create":
+		run("create registry", "k3d", "registry", "create", "registry.local", "--port", "12345")
+		run("create k8s cluster", "k3d", "cluster", "create", "local", "--registry-use", "k3d-registry.local:12345")
+		run("switch k8s context", "kubectl", "config", "use-context", "k3d-local")
+	// build and upload image to local registry
+	case "build":
+		context := "../../../../chainlink" // TODO: make this a flag
+		run("build image", "docker", "build", "-f", context+"/core/chainlink.Dockerfile", context, "-t", "chainlink:local")
+		run("tag image", "docker", "tag", "chainlink:local", "localhost:12345/chainlink:local")
+		run("push image", "docker", "push", "localhost:12345/chainlink:local")
+	// run ginkgo commands to spin up environment
+	case "run":
+		os.Chdir("../../../") // move to repo root
+		run("start environment", "ginkgo", "-r", "--focus", "@ocr", "integration-tests/smoke", "--", "--use-local-image")
+	// stop k8s namespace from environment
+	case "stop":
+		if len(os.Args) < 3 {
+			panic("missing namespace argument")
+		}
+		run("stopping environment", "kubectl", "delete", "namespaces", os.Args[2])
+	// delete removes the k8s cluster
+	case "delete":
+		run("remove k8s cluster", "k3d", "cluster", "delete", "local")
+		run("remove registry", "k3d", "registry", "delete", "k3d-registry.local")
+	default:
+		panic("unrecognized command")
+	}
+}
+
+func run(name string, f string, args ...string) {
```

Comment:
Let's move this to commons somewhere: https://github.com/smartcontractkit/chainlink-starknet/pull/41#discussion_r935711863

Reply:
I actually think this whole file could be extracted to potentially the `relay/ops` folder since there's nothing specific to starknet at the moment

I don't know if it needs to be extracted yet, but I will leave a note about the whole file